stack.yaml

```diff
@@ -3,3 +3,9 @@ resolver: ./snapshot.yaml
 packages:
   - .
   - test
+  # Library has never been released on Hackage yet
```

Comment:
We're adding stuff like this to the `snapshot.yaml` in this repo so they can be used in all repos. It should also be there already, so if you need something new just update the commit there.

Reply:
Gotcha